src/wasm-binary.h

```diff
@@ -888,8 +888,11 @@ enum ASTNodes {
   F32x4ConvertSI32x4 = 0xfa,
   F32x4ConvertUI32x4 = 0xfb,
 
-  F32x4QFMA = 0xfc,
-  F32x4QFMS = 0xfd,
+  V128Load32Zero = 0xfc,
+  V128Load64Zero = 0xfd,
+
+  F32x4QFMA = 0xb4,
```

Comment:
`0xb4` already uses for `I32x4DotSVecI16x8`:
https://github.com/WebAssembly/binaryen/blob/master/src/wasm-binary.h#L836

Is it correct?